app/controllers/api/generic_object_definitions_controller.rb

```diff
@@ -111,7 +111,7 @@ def self.allowed_types
     end
 
     def options
-      render_options(:generic_object_definitions, build_generic_object_definition_options)
+      render_options(@req.collection, build_generic_object_definition_options)
```

Comment:
Sorry.
these changes were not necessary here

I was looking to generalize the handling of `options` and these 3 changes came out here.
I decided this to be as far as we needed to go.

Reply:
@kbrock was this necessary?